tests/by-util/test_printenv.rs

```diff
@@ -40,3 +66,35 @@ fn test_invalid_option_exit_code() {
         .stderr_contains("unexpected argument")
         .stderr_contains("For more information, try '--help'");
 }
+
+#[test]
+fn test_null_separator() {
+    // printenv should use \x00 as separator
+    new_ucmd!()
+        .env("HOME", "FOO")
+        .env("KEY", "VALUE")
+        .arg("-0")
+        .succeeds()
+        .stdout_contains("HOME=FOO\x00")
+        .stdout_contains("KEY=VALUE\x00");
+
+    new_ucmd!()
+        .env("HOME", "FOO")
+        .env("KEY", "VALUE")
+        .arg("--null")
+        .succeeds()
+        .stdout_contains("HOME=FOO\x00")
+        .stdout_contains("KEY=VALUE\x00");
```

Comment:
Here it might make sense to use a loop over the args so it's easy to see that `-0` and `--null` are the same.

Reply:
Fixed. I wrapped it with a loop of `["-0", "--null"]`. Thanks for the review